### PR TITLE
Update runtime and patch it to build

### PR DESCRIPTION
**Pull request by lionirdeadman**

### CODE REVIEW

**Comment by lionirdeadman** [comment]
cc @hfiguiere

File: feathery-build.patch [hunk 1]
```diff
@@ -0,0 +1,58 @@
+From 5a3520540276a6fd06f7c88e571b6462978e3eab Mon Sep 17 00:00:00 2001
+From: Jammyjamjamman <jdsherratt3@gmail.com>
+Date: Sat, 20 Jun 2020 19:30:01 +0100
+Subject: [PATCH] Add ftp fixes
```
Comment (wjt) [inline feathery-build.patch:4]: Where did this patch come from?

File: flathub.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "skip-icons-check": true
```
Comment (wjt) [inline flathub.json:2]: I don't like it! Just skipping the check is not a solution, IMO.

(I accept that this is a preexisting problem, #2.)

**Review by wjt — COMMENTED** [review]
I don't have commit access to this repo, so for the purposes of review I'm just some guy.

19.08 is still supported. (The reason I updated it in #5 is that it was previously on 1.6, and it was not installable with modern flatpaks!) Is there any urgency in updating to 20.08?

**Comment by hfiguiere** [comment]
> I don't have commit access to this repo, so for the purposes of review I'm just some guy.
> 

Github was putting you in the reviewer list for some reason. I just didn't want to overstep. I do have merge permissions.

File: flathub.json (hunk 2 above)
Comment (hfiguiere) [reply flathub.json:2]: if this is not done then it'll never get merged.

File: feathery-build.patch (hunk 1 above)
Comment (hfiguiere) [reply feathery-build.patch:4]: This: https://github.com/MegaGlest/megaglest-source/commit/5a3520540276a6fd06f7c88e571b6462978e3eab

**Comment by lionirdeadman** [comment]
> 19.08 is still supported. (The reason I updated it in #5 is that it was previously on 1.6, and it was not installable with modern flatpaks!) Is there any urgency in updating to 20.08?

I'm just taking some advance before it EOLs because otherwise it'll be more work at once. I calculate that over 200 applications will need to be updated close to when 19.08 (and gnome 3.36) releases, @wjt.

**Pull request merged**